ufc/tests/test-case-flag-with-empty-string.json

```diff
@@ -0,0 +1,143 @@
+{
+  "flag": "string-variations",
+  "variationType": "STRING",
+  "defaultValue": {},
```

Comment:
default value can't be an empty object for STRING type (nor can it be null). This is why at least a couple of the tests are failing.
You'll probably want a non-empty string such as `"default-value"` to differentiate the default return from a proper blank string return.